run.sh

```diff
@@ -229,3 +235,6 @@ namespace_args=`kubectl get pods -o name |grep -w $RELEASE`
 
 echo -e "${BLUE}Check the logs by executing:${NC} kubectl logs ${namespace_args}"
 echo -e "${BLUE}Uninstall by executing:${NC} helm uninstall ${RELEASE}"
+
+APP_NAME="pythonservice" ./check_pods.sh
```

Comment:
Don't you need to set NAMESPACE here as well? 

Could you also set RELEASE to make it explicit?